### PR TITLE
build: add namespaces to makefile commands

### PR DESCRIPTION
**Pull request by xopham**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,13 +22,11 @@ install:
...
 uninstall:
-	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm uninstall connaisseur
+	helm uninstall connaisseur -n $(NAMESPACE)
 	kubectl delete ns $(NAMESPACE)
 
 upgrade:
-	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm upgrade connaisseur helm --wait
+	helm upgrade connaisseur helm  -n $(NAMESPACE)--wait
```
Comment (Starkteetje) [inline Makefile:29]: Do we also want to fix the NS in the integrations tests (ie https://github.com/sse-secure-systems/connaisseur/blob/master/tests/integration/integration-test.sh#L15 ) while you're at it?

Comment (xopham) [reply Makefile]: even introducing make as well to test both

File: tests/integration/integration-test.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ rm update
 echo 'Config set'
 
 echo 'Installing Connaisseur...'
-helm install connaisseur helm --atomic || { echo 'Failed to install Connaisseur'; exit 1; }
+make install || { echo 'Failed to install Connaisseur'; exit 1; }
```
Comment (Starkteetje) [inline tests/integration/integration-test.sh:15]: If you conciously choose a different installation method here that differs from the other ITs, I'd add a comment stating this explicitly

Comment (Starkteetje) [reply tests/integration/integration-test.sh:15]: But I like using both make and helm 👍

**Review by Starkteetje — APPROVED** [review]
Nice

**Pull request merged**